ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
@b49020 yeah I should have written `=` instead of `?=`. OK, let's not over-engineer things, now it seems to me your first proposal is valid for the uses cases we want to deal with. May I suggest that you add a comment, such as the following:
```
# Prevent use of LDFLAGS from the environment. For example, yocto exports
# LDFLAGS that are suitable for the client applications, not for TAs
LDFLAGS=
```
@jbech-linaro what do you think? I am OK with the `--no-builtin-variables` flag BTW.